tests/System/expected/test_webContext__TagManager.getAvailableTagTypesInContext.xml

```diff
@@ -1034,7 +1034,7 @@
 						<uiControlAttributes>
 						</uiControlAttributes>
 						<availableValues />
-						<description>(optional) the version of your project</description>
+						<description>(optional) the version of your project.</description>
```

Comment:
I really wonder about adding a `.` in cases like this where the description isn't even a full sentence.